packages/authentication/docs/authentication-system.md

```diff
@@ -163,3 +163,192 @@ And the abstractions for:
     - return user
   - controller function:
     - process the injected user
+
+## Registering an authentication strategy via an extension point
```

Comment:
I hope in follow-up PRs we can move this awesome doc to the README.

Reply:
@b-admike , yes. @jannyHou 's PR : https://github.com/strongloop/loopback-next/pull/2822 overlaps with mine, and we have discussed that we will be moving all separate .md files in the authentication package into a comprehensive readme.md file. And this readme file will also be the source of the loopback.io docs that are generated by our doc build process.